rust/crd/src/history.rs

```diff
@@ -24,7 +24,7 @@ use stackable_operator::{
     kube::{runtime::reflector::ObjectRef, ResourceExt},
     product_config_utils::{
         transform_all_roles_to_config, validate_all_roles_and_groups_config, Configuration,
-        ValidatedRoleConfigByPropertyKind,
+        Error as ConfigError, ValidatedRoleConfigByPropertyKind,
```

Comment:
nit: I'm not a big fan of import renames. Maybe use `product_config_utils::{self, ...}` in combination with `product_config_utils::Error` or fully qualified names instead.

Reply:
I was inspired by the similar change in Druid: https://github.com/stackabletech/druid-operator/blob/5b48d66852f1cbad4980b240e5550fa317f4c874/rust/crd/src/lib.rs#L40

I didn't think too much about it but it looks like using this alias makes it more future proof with respect to refactoring and restructurings of operator-rs in the future.

I would leave them as they are for now.

Reply:
I can see that maintainability might improve a little, but I just don't like the fact we are hiding the actual type behind an alias.

Also I wanted to avoid that we have two different styles of imports across our operators, because other PRs use fully qualified names or the trick I mentioned in my first comment. But if Druid uses it, I mostly fine with leaving it as is.